src/DynamoCore/Models/HomeWorkspaceModel.cs

```diff
@@ -116,6 +116,12 @@ protected override void OnNodeRemoved(NodeModel node)
             EngineController.NodeDeleted(node);
         }
 
+        protected override void ResetWorkspaceCore()
+        {
+            // Reset Run Automatic option to false on resetting the workspace
+            DynamicRunEnabled = false;
+        }
+
```

Comment:
@Benglin if we don't set this, when we have an existing workspace open with 'DynamicRunEnabled' set to 'true', and we open a new empty workspace, the new workspace continues to have the flag set to 'true'. I wanted to make it consistent:
Open file => flag turned off
Open new workspace => flag turned off